docs/public/configuration/gui/configuring-using-config-ts.md

```diff
@@ -0,0 +1,117 @@
+# Simple build-time configuring Zonemaster-GUI using "config.ts"
+
+The `config.ts` configuration file can be used for *build-time* settings of the
+Zonemaster-GUI, i.e. the installation package has to be rebuilt.
+
+## Finding "config.ts"
+
+In the source tree the file is found as `config.ts` relative to the root of the
+Git tree. It does not exist in the installed GUI, as the values have been built
+in
```

Comment:
```suggestion
In the source tree the file is found as `config.ts` relative to the root of
the Git tree. The installed GUI does not ship with this file, because the
values defined in this file are baked in the GUI’s HTML and JavaScript files.
```